frontend/catalyst/utils/toml.py

```diff
@@ -48,6 +53,25 @@
     from tomlkit.exceptions import TOMLKitError as TOMLException
 
 
+package_root = os.path.dirname(__file__)
+
+
+# Default paths to dep libraries
+DEFAULT_LIB_PATHS = {
+    "llvm": os.path.join(package_root, "../../../mlir/llvm-project/build/lib"),
+    "runtime": os.path.join(package_root, "../../../runtime/build/lib"),
+    "enzyme": os.path.join(package_root, "../../../mlir/Enzyme/build/Enzyme"),
+    "oqc_runtime": os.path.join(package_root, "../../catalyst/oqc/src/build"),
+}
```

Comment:
I don't see what this has to do with the toml system

Reply:
I moved it here because of the dependency chain: `get_lib_path` called by `get_device_toml_config` called by `get_device_capabilities`. The last function eventually creates `DeviceCapabilities` which is defined in the toml file. Do you have any suggestions?

Reply:
The toml files are also on some of those directories, but most are shared library locations we want to link. So I would not move these to the toml module

Reply:
Keeping these paths in the runtime.py creates a circular dependency. Do you have suggestions on how to solve it? Do you want me to create a separate `utils/paths.py` ? Please let me know

Reply:
After moving code I found that runtime.py contains only a few lines of code, so I renamed it into paths.py.

Reply:
Later renamed to runtime_environment as suggested